bin/release-cleanup

```diff
@@ -118,6 +117,8 @@ Please manually resolve conflicts in this file and continue when finished. Keep
 
   // Bump to next minor alpha version.
   execSync('npm run release:bump -- --yes');
+
+  console.log(chalk.magenta('Last step: manually commit and push the changes!'));
```

Comment:
If this was necessary, makes me think something went wrong with the `release:bump` command. Probably means that this instruction needs to happen _before_ `release:bump` in order for it to work.

Reply:
Hmm gotcha. I can take that line out. If it helps at all, this is what happened when I ran the script:

First time, it failed because my `main` branch wasn't up to date. I manually updated it, then reran the script, where this was the output:
```
? Welcome to the Irving release wizard! Which release branch would you like to merge? release/6.9
Switched to branch 'release/6.9'
Switched to branch 'main'
npx: installed 52 in 2.17s
npm-merge-driver: merging packages/integrations/package-lock.json
npm ERR! code EJSONPARSE
npm ERR! file /Users/amyevans/broadway/www/irving/packages/integrations/package.json
npm ERR! Merge conflict detected in your package.json.
npm ERR! 
npm ERR! Please resolve the package.json conflict and retry the command:
npm ERR! 
npm ERR! $ /usr/local/bin/node /usr/local/bin/npm install --package-lock-only

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/amyevans/.npm/_logs/2021-08-02T14_21_39_789Z-debug.log
npm-merge-driver: --legacy enabled. Checking out --theirs and retrying merge.
npm-merge-driver: !!!SOME CHANGES MAY BE LOST!!!
npm ERR! code EJSONPARSE
npm ERR! file /Users/amyevans/broadway/www/irving/packages/integrations/package.json
npm ERR! Merge conflict detected in your package.json.
npm ERR! 
npm ERR! Please resolve the package.json conflict and retry the command:
npm ERR! 
npm ERR! $ /usr/local/bin/node /usr/local/bin/npm install --package-lock-only

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/amyevans/.npm/_logs/2021-08-02T14_21_41_006Z-debug.log
Command failed: npm install --package-lock-only
? Merge conflict encountered. Have any dependencies been installed in the release branch that have not been merged into main? No
? Unfortunately, the root CHANGELOG.md file cannot be automatically merged.
Please manually resolve conflicts in this file and continue when finished. Keep in mind:
- every changelog heading should have release notes, even if it's just "**Note:** Version bump only for package irving"
- depending on how far your release branch and the main branch have diverged, there may be duplicate changelog entries. Verify this is no
t the case before completing your merge. Yes
Bumping main branch to next version...
lerna notice cli v3.20.2
lerna info current version 6.9.0-alpha.3
lerna notice Current HEAD is already released, skipping change detection.
lerna success No changed packages to publish 
? What version are you releasing? (e.g. 6.5) 6.9
Commit roundup generated for release 6.9
### [6.9.0-alpha.3](https://github.com/alleyinteractive/irving/compare/v6.9.0-alpha.2...v6.9.0-alpha.3) (2021-07-29)
#### Bug Fixes
* **commentcountstatic:** differentiate class name for secondary Coral comment count component" ([646d822](https://github.com/alleyinteractive/irving/commit/646d822932fd91861fc1ad81caa51445e5252a36))
### [6.9.0-alpha.2](https://github.com/alleyinteractive/irving/compare/v6.9.0-alpha.1...v6.9.0-alpha.2) (2021-07-29)
#### Bug Fixes
* **analytics:** fix [object object] ga event action ([53c6896](https://github.com/alleyinteractive/irving/commit/53c689627632be3fb822c30ad3a4917b9e55c8c7))
### [6.9.0-alpha.1](https://github.com/alleyinteractive/irving/compare/v6.9.0-alpha.0...v6.9.0-alpha.1) (2021-07-29)
#### Bug Fixes
* **commentcountstatic:** add support for additional Coral config in static component ([232f7c2](https://github.com/alleyinteractive/irving/commit/232f7c2c5558716e705e1af35d1c7c6b97f2d70e))
### [6.9.0-alpha.0](https://github.com/alleyinteractive/irving/compare/v6.8.1...v6.9.0-alpha.0) (2021-07-28)
#### Features
* **commentcountstatic:** add new Coral component for displaying static comment count data ([a89a8ac](https://github.com/alleyinteractive/irving/commit/a89a8ac35b38c1811f0dba68bf956a7e5ea66b1a))
## [6.8.1-alpha.0](https://github.com/alleyinteractive/irving/compare/v6.8.0...v6.8.1-alpha.0) (2021-07-23)
### Bug Fixes
* **logservice:** fix issue with latest verstion of `util` package referencing `process` ([d6f1562](https://github.com/alleyinteractive/irving/commit/d6f1562d89e0f939548a1b996f9cf5210bd63c79))
## [6.8.1-alpha.0](https://github.com/alleyinteractive/irving/compare/v6.8.0...v6.8.1-alpha.0) (2021-07-23)
**Note:** Version bump only for package irving
## [6.8.1](https://github.com/alleyinteractive/irving/compare/v6.8.1-beta.0...v6.8.1) (2021-07-23)
**Note:** Version bump only for package irving
## [6.8.1-beta.0](https://github.com/alleyinteractive/irving/compare/v6.8.0...v6.8.1-beta.0) (2021-07-23)
### Bug Fixes
* **logservice:** fix issue with latest verstion of `util` package referencing `process` ([d6f1562](https://github.com/alleyinteractive/irving/commit/d6f1562d89e0f939548a1b996f9cf5210bd63c79))

lerna notice cli v3.20.2
lerna info current version 6.9.0-alpha.3
lerna notice Current HEAD is already released, skipping change detection.
lerna success No changed packages to publish
```